package.json

```diff
@@ -90,6 +91,7 @@
     "ember-computed-decorators": "^0.2.2",
     "ember-cli-htmlbars": "^1.0.1",
     "ember-cli-sass": "5.6.0",
+    "ember-frost-core": "^1.0.0",
```

Comment:
I thought I put a note, maybe it wasn't in this PR, where the core install won't actually work, but I don't think it really matters for addons that depend on core. I don't think anyone is getting core by installing some other addon in their app. I think it's safe (for this transition period at least) to just tell people to install core and test independently.